rclc_parameter/include/rclc_parameter/rclc_parameter.h

```diff
@@ -340,7 +340,7 @@ rcl_ret_t
 rclc_parameter_get_int(
   rclc_parameter_server_t * parameter_server,
   const char * parameter_name,
-  int * output);
+  int64_t * output);
```

Comment:
This would break API compatability!

Reply:
I agree, but at some point, this can be considered a bug...

Reply:
Yes, I agree, but we must not break API or ABI stability within a release. We could add a static_cast to int64_t within the function as a bug fix (with would happen anyways implicitly ...)

Reply:
Do you think this will lead to a bug? I guess only in the rclc_parameter_get_int function, where the integer value (int64_t) is returned as (int). So we could add in the documentation and in the header file in Galactic release:
Limitation:
- maximum number allowed is sizeof(int)